src/pages/Home/components/HeroSection/components/newsletterLink/newsletter.module.scss

```diff
@@ -0,0 +1,93 @@
+.container {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  z-index: 1;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  width: 336px;
+  padding: 24px 32px;
+  background: linear-gradient(91.11deg,#1d333c 2.59%, #17282e 88.15%);
+  opacity: 0;
+  transform: translateY(20px);
+  animation: rise-up 0.6s ease-out 1.3s forwards;
+
+  @keyframes rise-up {
+    from {
+      opacity: 0;
+      transform: translateY(20px);
+    }
+
+    to {
+      opacity: 1;
+      transform: translateY(0);
+    }
+  }
+
+  &:hover {
+
+    .iconContainer {
+      background-color: #FFF;
+    }
+
+    .icon {
+      transform: translateX(3px);
+      fill: var(--common-color-text-primary);
+    }
+  }
+
+  @media (max-width: 900px){
+    visibility: hidden;
+  }
+}
+
+.content {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  font-size: 100%;
+  background: transparent;
+  flex: 1;
+}
+
+.header{
+  font-family: var(--common-font-en-bold), sans-serif;
+  font-size: 1.125rem;
+  font-weight: normal;
+  line-height: 1.5;
+  letter-spacing: 0.02em;
+}
+
+.text {
+  margin-top: 12px;
+  font-size: 0.875rem;
+  font-weight: normal;
+  line-height: 1.4;
+}
+
+.iconContainer {
```

Comment:
let's use lowercase for letters and underscore. naming like in BEM instead of camelcase